### PR TITLE
Release notes 2.6 added

### PR DESCRIPTION
**Pull request by mdeneva**
Please review the Release Notes. I assume release date is 14.02.2022 (I might be wrong)

### CODE REVIEW

**Comment by cannisondat** [comment]
Do not merge until we are officially releasing - need to coordinate with Janki

File: docs/reference/release_notes.md
Comment (Arau) [inline docs/reference/release_notes.md:38]: s/1.2.3/v1.23/

Btw is it tested that we support 1.23?

File: docs/reference/release_notes.md
Comment (Arau) [inline docs/reference/release_notes.md:34]: I would change this sentence because it is not only about limits, but also requests. Sth lik


Resource requests and resource limits defined for the Ondat different components (csi-attacher, csi-provisioner, csi-resizer, api-manager, cluster-operator and ondat-scheduler). 

I think adding the components is important because I'm not sure if the node manager and the portal manager have that set be default, which they absolutely should.

**Review by Arau — CHANGES_REQUESTED** [review]

File: docs/reference/release_notes.md
Comment (cannisondat) [reply docs/reference/release_notes.md:38]: it is being tested right now - @DavidMarchant fyi

File: docs/reference/release_notes.md
Comment (cannisondat) [inline docs/reference/release_notes.md:31]: Add the following:

- We have added capability for airgapped installation. The new options can also be used outside of an airgapped cluster.

**Review by cannisondat — CHANGES_REQUESTED** [review]
Please add a new line regarding airgapped installation capability in kubectl plugin - thank you !

File: docs/reference/release_notes.md
Comment (cannisondat) [inline docs/reference/release_notes.md:30]: New line between the sentences?

**Review by cannisondat — APPROVED** [review]
lgtm - approved but **please do not merge until confirmed with marketing**

File: docs/reference/release_notes.md
Comment (Arau) [reply docs/reference/release_notes.md:39]: @mdeneva I'd be happy to approve and merge when the v 1.2.3 is changed to v1.23

**Comment by Arau** [comment]
Changed base to merge against [release/v2.6.0](https://github.com/ondat/documentation/tree/release/v2.6.0) branch instead of main.

**Comment by cannischan** [comment]
@Arau please check today so we are ready for publishing tomorrow, thanks!

**Review by Arau — APPROVED** [review]

**Pull request merged**
